src/PanZoom.tsx

```diff
@@ -44,17 +46,19 @@ const PanZoom: React.FC<PanZoomProps> = ({
   }, [className, selecting]);
 
   const childStyle: React.CSSProperties = useMemo(() => {
-    let style = {
+    let style: React.CSSProperties = {
       ...CHILD_STYLE,
       height,
       width,
     };
 
-    if (disabledUserSelect) style = { ...style, ...CHILD_DISABLED_STYLE };
+    if (disabledUserSelect)
+      style = { ...style, ...CHILD_DISABLED_STYLE } as React.CSSProperties;
     if (selecting) style.pointerEvents = 'all';
+    if (cover) style.backgroundImage = 'url(' + cover + ')';
```

Comment:
I was thinking about that a lot and I think it should be more clean to adding stuff like that directly in CSS

Reply:
agree